### ISSUE TITLE
Calculate earnings, baseline_net, and reform_net only once in situations/charts.py

### ISSUE THREAD

**Issue by MaxGhenis** [PolicyEngine/policyengine#116]
Currently it calculates these once in the net income chart and once in the MTR chart. If these charts could share the data, it could improve performance.

**Comment by MaxGhenis** [comment]
Sounds good, closing as wontfix.
